### PR TITLE
Fix binding redirect after recent package updates

### PR DESCRIPTION
**Pull request by erdembayar**
Recent many packages update broke our binding redirect.

### CODE REVIEW

File: src/NuGetGallery/Web.config [hunk 1]
```diff
@@ -584,6 +584,14 @@
   </system.diagnostics>
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+	  <dependentAssembly>
+	  	<assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="CC7B13FFCD2DDD51" culture="neutral"/>
```
Comment (joelverhagen) [inline src/NuGetGallery/Web.config:588]: super nit: consistent spacing

File: src/NuGetGallery/Web.config [hunk 2]
```diff
@@ -584,6 +584,14 @@
   </system.diagnostics>
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+	  <dependentAssembly>
```
Comment (joelverhagen) [inline src/NuGetGallery/Web.config:587]: Were these generated by VS?

Comment (erdembayar) [reply src/NuGetGallery/Web.config:587]: Yes, VS generated for me.

File: src/NuGetGallery/Web.config (hunk 1 above)
Comment (erdembayar) [reply src/NuGetGallery/Web.config]: Corrected, looks it was `space` vs `tab` problem.

File: src/NuGetGallery/Web.config [hunk 3]
```diff
@@ -584,6 +584,13 @@
   </system.diagnostics>
   <runtime>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="CC7B13FFCD2DDD51" culture="neutral"/>
+        <bindingRedirect oldVersion="0.0.0.0-5.0.0.0" newVersion="5.0.0.0"/>
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="NuGet.Frameworks" publicKeyToken="31BF3856AD364E35" culture="neutral"/>
```
Comment (joelverhagen) [inline src/NuGetGallery/Web.config:592]: Is this one missing a `<bindingRedirect>` element?

**Comment by erdembayar** [comment]
Closing in favor of https://github.com/NuGet/NuGetGallery/pull/9596/files

**Pull request closed without merging**